SparkPost/2web2ui
# SA-1847: Update form and tests

### What Changed
 - Added timezone and schedule_type as request params.
 - Changed logic for formatting form values due to new cron format.

### How To Test
 - Make sure `allow_scheduled_reports` account ui flag is enabled.
 - Go to the create scheduled page. EX: /signals/schedule/a601ccd1-49da-4c8f-bad1-e16e77302fbc. OR from the list report modal in report builder and select schedule in the action list. 
 - Check that timezone and schedule_type is added correctly.
 - Check that cronjob formatting works correctly.
